specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/2015-06-15/Quotas.json

```diff
@@ -0,0 +1,310 @@
+            "get": {
+                "x-ms-examples": {
+                    "Get a quota by name": {
+                        "$ref": "./examples/Quota/Get.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "description": "Get a quota by name.",
+                "operationId": "Quotas_Get",
+                "parameters": [
+                    {
+                        "$ref": "Network.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ResourceNameParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Quota"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found"
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create or update a new quota": {
+                        "$ref": "./examples/Quota/Put.json"
+                    }
+                },
+                "tags": [
+                    "Network"
+                ],
+                "description": "Create or update a quota.",
+                "operationId": "Quotas_CreateOrUpdate",
+                "parameters": [
+                    {
+                        "$ref": "Network.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ResourceNameParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/QuotaParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Quota"
+                        }
+                    },
+                    "201": {
+                        "description": "Created",
+                        "schema": {
+                            "$ref": "#/definitions/Quota"
+                        }
+                    }
```

Comment:
If this is a long running operation then please mark so e.g. https://github.com/Azure/azure-rest-api-specs/blob/a93922b09f8ccba9899f8dc8c3cc9de6e5b76776/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json#L533

Reply:
It isn't.  Long running operations should return 202 and then eventually 200/201 correct?